train.py

```diff
@@ -632,16 +631,13 @@ def run(cfg: Any) -> None:
     parser.add_argument(
         "-C", "--config", help="config filename", default=argparse.SUPPRESS
     )
-    parser.add_argument(
-        "-P", "--pickle", help="pickle filename", default=argparse.SUPPRESS
-    )
+    parser.add_argument("-Y", "--yaml", help="yaml filename", default=argparse.SUPPRESS)
     parser_args, unknown = parser.parse_known_args(sys.argv)
 
     if "config" in parser_args:
-        cfg = load_config(parser_args.config)
-    elif "pickle" in parser_args:
-        with open(parser_args.pickle, "rb") as pickle_file:
-            cfg = dill.load(pickle_file)
+        cfg = load_config_py(parser_args.config)
```

Comment:
mark as deprecated? or should we keep it?

Reply:
We can remove it once the kaggle/colab kernels have been updated

Reply:
Sounds good to have the option. Let's keep it then